docs/slurk_multibots.rst

```diff
@@ -3,4 +3,90 @@
 Pairing up participants and running multiple rooms (and bots)
 =============================================================
 
-*Explain how to run the waiting room / pairup bot, and the multibot.*
+In the following steps you will learn how to let bots and human users enter the slurk waiting room where users will be paired up and put into a new task-specific chatroom along with bots. The bot handling this mechanism (moving human users and other bots from the waiting room to the taskroom) is the **pairup bot**. You can find it in the *sample_bots* directory.
```

Comment:
```suggestion
In the following steps you will learn how to let bots and human users enter the slurk waiting room where users will be paired up and put into a new task-specific chatroom along with bots. We provided an example bot handling this mechanism: The **pairup bot** moves human users and other bots from the waiting room to a taskroom. You can find it in the *sample_bots* directory.
```

Reply:
We should explicitly state, that **pairup bot** is an example bot. This was also missinterpreted by many of the WG members: It just demonstrates, **how** moving users is working. This is btw. one of the reasons we want to deprecate the sample bots directory in the next version and remove it entirely in 2.0. cc #45